source/index.html.md

```diff
@@ -14,7 +14,341 @@ Through our SSO integration HomeSpotter powers a Boost check out process enablin
 
 The Boost check out process allows the user to customize a pre-populated creative, confirm targeting, confirm budget and launch a campaign.
 
-# Authentication
+# Boost Integration V2
+Integration intended for partners without knowledge of current Boost customers.
```

Comment:
Is it worth calling out any shortcomings of the V2 integration approach?  I'd imagine the biggest issue is that, since there's no "Sign In" button, this won't let users leverage their existing brokerage-provided account if they have one, right?  Would that, or any other reason, be a compelling argument for a partner to use the V1 API?

Reply:
I would assume we will be starting the sign in button work in the next month.  It is prioritized next.